modules/special-rules/pages/flak.adoc

```diff
@@ -1,2 +1,62 @@
 = Flak
 
+These units are specialised to shoot down enemy flyers and drop pods.
+Some use very sophisticated targeting devices and fast, accurate weapons systems.
+Others simply fill the air with so many munitions that they can hardly fail to get some hits.
+
+---
+
+{flak} units can attack flyers in one of two different ways, depending on what orders you give to their detachment.
+
+[IMPORTANT]
```

Comment:
```suggestion
[NOTE]
```
One has to be careful not to overdo notices, and to use the right notice type for the 'level of attention' that you need to grab. Overuse or misuse devalues them and they become 'noise' to the reader. 
IMPORTANT is meant for things that are all too easy to forget at an important moment, like the need to take a Leadership test before you can place certain orders (or move, or whatever it is, I forget!). 
I should probably try to codify usage in the Contributor's Guide. But this is the rough idea.

Reply:
Yes, that definitely needs adding to the guide. I'm used to using IMPORTANT for marking important parts of a rule that aren't in the main bullet points, and think of NOTE as being more like a reminder, so it wouldn't occur to me to use it as a wrapper for primary rules content.